meta-ivi/recipes-support-ivi/keyutils/keyutils_1.5.9.bb

```diff
@@ -16,6 +16,7 @@ SRC_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git;p
           file://keyutils-fix-the-cflags-for-all-of-targets.patch \
           "
 SRC_URI_append_arm = " file://keyutils-arm-remove-m32-m64.patch"
+SRC_URI_append_aarch64 = " file://keyutils-arm-remove-m32-m64.patch"
 SRC_URI_append_x86 = " file://keyutils_fix_x86_cflags.patch"
 SRC_URI_append_x86-64 = " file://keyutils_fix_x86-64_cflags.patch"
 SRC_URI_append_powerpc = "file://keyutils-fix-powerpc-cflags.patch"
```

Comment:
@tolkien-joh 
Another SRC_URI_append has machine dependency. So should I move another SRC_URI_appends? or only my patch?

Thanks